server.py

```diff
@@ -0,0 +1,71 @@
+import socket
+import threading
+from data import load_data, get_albums, get_album_songs, get_song_length, get_song_lyrics, get_song_album, search_songs_by_name, search_songs_by_lyrics
+
+#משתנים קבועים
+SERVER_IP = '127.0.0.1'
+SERVER_PORT = 12345
+BUFFER_SIZE = 1024
+
+#טעינת הנתונים
+FILE_PATH = 'Pink_Floyd_DB.txt'
+albums, songs_info = load_data(FILE_PATH)
+
+#יצירת הסוקט והקשורות לשרת עד 5 לקוחות
+server_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+server_socket.bind((SERVER_IP, SERVER_PORT))
+server_socket.listen(5)
+
+print(f"Server is listening on {SERVER_IP}:{SERVER_PORT}...")
+
+#פונקציה לטיפול בלקוחות
+def handle_client(client_socket):
+    client_socket.sendall(b"Welcome to the Pink Floyd Server!\n")
+    
+    while True:
+        request = client_socket.recv(BUFFER_SIZE).decode('utf-8').strip()
+        if not request:
+            break
+
+        print(f"Received request: {request}")
+
+        response = ""
+        if request == "1":
```

Comment:
you should use a `match` block
this function also has a lot of repeating code, make sure to minimize it